<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-09-26T08:29:59Z" name="Default suite" finished-at="2021-09-26T08:32:42Z" duration-ms="162386">
    <groups>
    </groups>
    <test started-at="2021-09-26T08:29:59Z" name="Default test" finished-at="2021-09-26T08:32:42Z" duration-ms="162386">
      <class name="com.chekinckingLinks.test.CheckLinksTest">
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:com.chekinckingLinks.test.CheckLinksTest@45099dd3]" started-at="2021-09-26T03:29:59Z" name="beforeClass" finished-at="2021-09-26T03:30:04Z" duration-ms="4785" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="checkingList()[pri:0, instance:com.chekinckingLinks.test.CheckLinksTest@45099dd3]" started-at="2021-09-26T03:30:04Z" name="checkingList" finished-at="2021-09-26T03:32:42Z" duration-ms="157535" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[There are broken list expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: There are broken list expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.chekinckingLinks.test.CheckLinksTest.checkingList(CheckLinksTest.java:30)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkingList -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:com.chekinckingLinks.test.CheckLinksTest@45099dd3]" started-at="2021-09-26T03:32:42Z" name="afterClass" finished-at="2021-09-26T03:32:42Z" duration-ms="52" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- com.chekinckingLinks.test.CheckLinksTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
